src/apps/auth/data-access/invitation/invitation.repository.ts

```diff
@@ -0,0 +1,27 @@
+import { InvitationModel } from "./invitation.model";
+import { IInvitation } from "./invitation.interface";
+import { Types } from "mongoose";
+
+export class InvitationRepository {
+  async create(invitationData: Partial<IInvitation>): Promise<IInvitation> {
```

Comment:
ici tu peux hériter de MongooseRepository pour avoir les méthodes communes comme create. et tu peux accéder au modèle directement en faisant `this.model` si tu en as besoin pour les méthodes custom

Reply:
ok